bundles/framework/announcements/view/AnnouncementsCollapse.jsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import PropTypes from 'prop-types';
+import { Controller, LocaleConsumer } from 'oskari-ui/util';
+import { Message, Collapse, CollapsePanel } from 'oskari-ui';
+import { AnnouncementsModal } from './AnnouncementsModal';
+import { Divider } from 'antd';
+
+//Collapse panel -> set title, content and date range according to announcement
+
+const { Panel } = Collapse;
+
+const AnnouncementsCollapse = ({controller, updated, panels, modals, checked }) => {
+
+      if (!updated) {
+        controller.getAnnouncements(function(ann){
+          console.log("get announcements collapse");
+          console.log(ann);
+
+        
+          var panels = [];
+          var modals = [];
+            ann.data.forEach((announcement) => {
+
+              if (announcement.active && controller.showModal(announcement.id)) {
```

Comment:
Toggle the active flag in correct state on the controller before we hit render here so you don't have to double-check with controller.showModal()

Reply:
Have to check showModal to see is the announcement added to localStorage ie. user checked it as 'don't show', or am I understanding this wrong?